package.json

```diff
@@ -2,92 +2,76 @@
   "name": "cdp-node-backend-template",
   "version": "0.1.0",
   "description": "CDP Backend Template",
-  "main": ".server/index.js",
+  "main": "src/index.js",
   "type": "module",
   "engines": {
     "node": ">=22"
   },
   "scripts": {
-    "build": "npm run build:server",
-    "build:server": "NODE_ENV=production babel --delete-dir-on-start --ignore \"**/*.test.js\" --ignore \"**/__fixtures__\" --ignore \"**/test-helpers\" --copy-files --no-copy-ignored --out-dir ./.server ./src",
     "docker:dev": "NODE_ENV=development npm run server:watch",
     "dev": "npm run server:watch",
     "dev:debug": "npm run server:debug",
-    "format": "prettier --write \"src/**/*.js\" \"**/*.{js,cjs,md,json,config.js,test.js,graphql.js}\"",
-    "format:check": "prettier --check \"src/**/*.js\" \"**/*.{js,cjs,md,json,config.js,test.js,graphql.js}\"",
-    "git:pre-commit-hook": "npm run format:check && npm run lint && npm test",
+    "format": "prettier --write \"**/*.{md,json}\"",
+    "format:check": "prettier --check \"**/*.{md,json}\"",
+    "git:pre-commit-hook": "npm run format:check && npm run lint",
     "postinstall": "npm run setup:husky",
-    "lint": "run-s lint:js lint:types",
-    "lint:fix": "eslint --cache . --fix",
-    "lint:js": "eslint --cache .",
-    "lint:types": "tsc --build tsconfig.json",
+    "lint": "standard",
+    "lint:fix": "standard --fix",
     "postversion": "git add package.json package-lock.json && git commit -m $npm_package_version",
-    "test": "jest --coverage --verbose --runInBand",
-    "test:watch": "jest --watch",
-    "server:watch": "nodemon --exec tsx --enable-source-maps ./src",
-    "server:debug": "nodemon --exec tsx --enable-source-maps --inspect ./src",
-    "prestart": "npm run build",
+    "test": "node --no-experimental-require-module ./node_modules/jest/bin/jest.js --coverage --verbose --runInBand",
```

Comment:
`--no-experimental-require-module` is needed as Node 22 changes the behaviour of `require` which in turn breaks `@shelf\jest-mongodb`

https://github.com/nodejs/node/releases/tag/v22.12.0